packages/graphics/libdrm/package.mk

```
@@ -48,11 +48,14 @@ listcontains "${GRAPHIC_DRIVERS}" "etnaviv" &&
 
 post_makeinstall_target() {
   # Remove all test programs installed by install-test-programs=true except modetest
-  # Do not "not use" the ninja install and replace this with a simple "cp modetest"
-  # as ninja strips the unnecessary build rpath during the install.
-  safe_remove ${INSTALL}/usr/bin/amdgpu_stress
-  safe_remove ${INSTALL}/usr/bin/drmdevice
-  safe_remove ${INSTALL}/usr/bin/modeprint
-  safe_remove ${INSTALL}/usr/bin/proptest
-  safe_remove ${INSTALL}/usr/bin/vbltest
+
+  for PKG_LIBDRM_TEST in \
+    drmdevice modeprint proptest vbltest
+  do
+    safe_remove ${INSTALL}/usr/bin/${PKG_LIBDRM_TEST}
+  done
```

```suggestion
  for PKG_LIBDRM_TEST in drmdevice modeprint proptest vbltest; do
    safe_remove "${INSTALL}/usr/bin/${PKG_LIBDRM_TEST}"
  done
```

not tested, but looks better ?

This is likely not working ?
`safe_remove ${INSTALL}/usr/bin/{drmdevice,modeprint,proptest,vbltest}`

---

The actual code is copy past of one of the get plugins packages & IMHO I prefer readability over short "one liners"